src/app/checkout/step-3/step-3.component.js

```diff
@@ -175,6 +176,7 @@ class Step3Controller {
         error.config.data['security-code'] = error.config.data['security-code'].replace(/./g, 'X') // Mask security-code
       }
       componentInstance.$log.error('Error submitting purchase:', error)
+      datadogRum.addError(new Error(`Error submitting purchase: ${JSON.stringify(error)}`)) // here in order to show up in Error Tracking in DD
```

Comment:
I opted to not tweak the test that checks `$log.error` because I found it not worth the time (mocking was not playing nicely).

Reply:
I think the main thing here is that we test to ensure we get an error when an issue arises, and the test does that.